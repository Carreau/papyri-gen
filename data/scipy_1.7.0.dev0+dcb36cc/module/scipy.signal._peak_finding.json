{
  "_content": {
    "Summary": {
      "children": [
        {
          "type": "Paragraph",
          "data": {
            "children": [
              {
                "type": "Words",
                "data": {
                  "value": "Functions for identifying peaks in signals."
                }
              }
            ]
          }
        }
      ]
    },
    "Extended Summary": {
      "children": []
    },
    "Parameters": {
      "children": []
    },
    "Returns": {
      "children": []
    },
    "Yields": {
      "children": []
    },
    "Receives": {
      "children": []
    },
    "Raises": {
      "children": []
    },
    "Warns": {
      "children": []
    },
    "Other Parameters": {
      "children": []
    },
    "Attributes": {
      "children": []
    },
    "Methods": {
      "children": []
    },
    "Notes": {
      "children": []
    },
    "Warnings": {
      "children": []
    }
  },
  "refs": [],
  "ordered_sections": [
    "Summary",
    "Extended Summary"
  ],
  "item_file": "/Users/bussonniermatthias/dev/scipy/scipy/signal/_peak_finding.py",
  "item_line": 0,
  "item_type": "<class 'module'>",
  "aliases": [
    "scipy.signal._peak_finding"
  ],
  "example_section_data": {
    "children": []
  },
  "see_also": [],
  "signature": null,
  "references": null
}