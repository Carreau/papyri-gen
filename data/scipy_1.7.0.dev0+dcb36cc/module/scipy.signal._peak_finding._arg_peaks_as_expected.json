{
  "_content": {
    "Summary": {
      "children": [
        {
          "type": "Paragraph",
          "data": {
            "children": [
              {
                "type": "Words",
                "data": {
                  "value": "Ensure argument "
                }
              },
              {
                "type": "Directive",
                "data": {
                  "value": [
                    "peaks"
                  ],
                  "domain": null,
                  "role": null
                }
              },
              {
                "type": "Words",
                "data": {
                  "value": " is a 1-D C-contiguous array of dtype('intp')."
                }
              }
            ]
          }
        }
      ]
    },
    "Extended Summary": {
      "children": [
        {
          "type": "Paragraph",
          "data": {
            "children": [
              {
                "type": "Words",
                "data": {
                  "value": "Used in "
                }
              },
              {
                "type": "Directive",
                "data": {
                  "value": [
                    "peak",
                    "_",
                    "prominences"
                  ],
                  "domain": null,
                  "role": null
                }
              },
              {
                "type": "Words",
                "data": {
                  "value": " and "
                }
              },
              {
                "type": "Directive",
                "data": {
                  "value": [
                    "peak",
                    "_",
                    "widths"
                  ],
                  "domain": null,
                  "role": null
                }
              },
              {
                "type": "Words",
                "data": {
                  "value": " to make "
                }
              },
              {
                "type": "Directive",
                "data": {
                  "value": [
                    "peaks"
                  ],
                  "domain": null,
                  "role": null
                }
              },
              {
                "type": "Words",
                "data": {
                  "value": " compatible with the signature of the wrapped Cython functions."
                }
              }
            ]
          }
        }
      ]
    },
    "Parameters": {
      "children": []
    },
    "Returns": {
      "children": [
        {
          "type": "Param",
          "data": {
            "param": "value",
            "type_": "ndarray",
            "desc": [
              {
                "type": "Paragraph",
                "data": {
                  "children": [
                    {
                      "type": "Words",
                      "data": {
                        "value": "A 1-D C-contiguous array with dtype('intp')."
                      }
                    }
                  ]
                }
              }
            ]
          }
        }
      ]
    },
    "Yields": {
      "children": []
    },
    "Receives": {
      "children": []
    },
    "Raises": {
      "children": []
    },
    "Warns": {
      "children": []
    },
    "Other Parameters": {
      "children": []
    },
    "Attributes": {
      "children": []
    },
    "Methods": {
      "children": []
    },
    "Notes": {
      "children": []
    },
    "Warnings": {
      "children": []
    }
  },
  "refs": [],
  "ordered_sections": [
    "Summary",
    "Extended Summary",
    "Returns"
  ],
  "item_file": "/Users/bussonniermatthias/dev/scipy/scipy/signal/_peak_finding.py",
  "item_line": 267,
  "item_type": "<class 'function'>",
  "aliases": [
    "scipy.signal._peak_finding._arg_peaks_as_expected"
  ],
  "example_section_data": {
    "children": []
  },
  "see_also": [],
  "signature": "_arg_peaks_as_expected(value)",
  "references": null
}